<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reg16Widget</class>
 <widget class="QWidget" name="Reg16Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>18</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="layout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>17</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>17</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">XX</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="frameLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="valueLabel">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 5px; padding-right: 5px</string>
        </property>
        <property name="text">
         <string notr="true">-32768</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionHexadecimalSigned">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hexadecimal / Signed</string>
   </property>
  </action>
  <action name="actionHexadecimalUnsigned">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>He&amp;xadecimal /Unsigned</string>
   </property>
  </action>
  <action name="actionDecimalSigned">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal / &amp;Signed</string>
   </property>
  </action>
  <action name="actionDecimalUnsigned">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Decimal / &amp;Unsigned</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
